query/common/src/main/java/org/codice/ddf/admin/common/services/ServiceCommons.java

```diff
@@ -64,11 +61,13 @@ public String resolveProperty(String str) {
     return Arrays.stream(list).map(this::resolveProperty).collect(Collectors.toList());
   }
 
+  @SuppressWarnings("squid:S1135" /* Remove when TODO is completed */)
   public Report<Void> createManagedService(Map<String, Object> serviceProps, String factoryPid) {
     Configurator configurator = configuratorSuite.getConfiguratorFactory().getConfigurator();
     configurator.add(configuratorSuite.getManagedServiceActions().create(factoryPid, serviceProps));
 
-    // TODO RAP 13 Jul 17: Blank out password in the parameters passed here
+    // TODO RAP 13 Jul 17: Blank out password in the parameters passed here. Remove sonar
```

Comment:
Do we even need to do this since we have the PasswordField?

Reply:
I was wondering that. Let me double check with Richard.

Reply:
This TODO is still valid. I'm gonna go ahead and create a ticket for it.